client/app/components/modal/CreateEventModal.jsx

```diff
@@ -160,6 +159,18 @@ export default class CreateEventModal extends React.Component {
     this.setState({ errors: {} });
   }
 
+  _clearForm() {
+    this.setState({
+      showErrorMessage: false,
+      summary: '',
+      description: '',
+      conferenceRoomId: null,
+      availableRooms: [],
+      unavailableRooms: [],
+      errors: {}
+    });
```

Comment:
Can we extract this to sth like deafultFilters so that we won't have duplicated code?